presto-orc/src/main/java/com/facebook/presto/orc/writer/SliceDictionaryColumnWriter.java

```diff
@@ -400,7 +402,9 @@ public long getBufferedBytes()
     @Override
     public long getRetainedBytes()
     {
-        return dataStream.getRetainedBytes() +
+        // NOTE: we do not include stats because they should be small and it would be annoying to calculate the size
+        return INSTANCE_SIZE +
+                dataStream.getRetainedBytes() +
```

Comment:
Please include `values`. It is among the core set of fields in this writer. It is also likely not small.